soapui-maven-plugin/project.xml

```diff
@@ -132,10 +132,15 @@
 			<type>jar</type>
 		</dependency>
 		<dependency>
-			<groupId>groovy</groupId>
+			<groupId>org.codehaus.groovy</groupId>
 			<artifactId>groovy-all</artifactId>
-			<version>${groovy-all.version}</version>
-			<type>jar</type>
+			<version>3.0.6</version>

```

Comment:
Why don't use ${groovy-all.version} here?

Reply:
project.xml doesn't have any links on parent project. It's unclear for me how it will work and from where this value "groovy-all.version" will be taken.